x-pack/plugins/synthetics/server/synthetics_service/synthetics_service.ts

```diff
@@ -528,7 +528,7 @@ export class SyntheticsService {
     if (paramsBySpace[ALL_SPACES_ID]) {
       Object.keys(paramsBySpace).forEach((space) => {
         if (space !== ALL_SPACES_ID) {
-          paramsBySpace[space] = Object.assign(paramsBySpace[space], paramsBySpace['*']);
+          paramsBySpace[space] = Object.assign(paramsBySpace[space], paramsBySpace[ALL_SPACES_ID]);
```

Comment:
I would think that space specific params should overwrite all spaces params... though... can you even create an all spaces param with the same name as a param in normal space... I might have to test this...

Reply:
So I tested this.

If you create a space-scoped param in one space (for example default), then navigate to another space (space-2) and create an ALL_SPACES param, then navigate back to the original space (default) it is possible to create two params with the same name. 

<img width="1410" alt="Screen Shot 2023-03-29 at 12 22 39 PM" src="https://user-images.githubusercontent.com/11356435/228604241-93022fda-35b0-4df8-b6e3-b74b0057bbd6.png">

Not sure if this is desired, but if so the space-scoped params should take precedence, I would think.